src/commands/imageSource/buildImageInAzure/BuildImageStep.ts

```diff
@@ -23,11 +24,16 @@ export class BuildImageStep extends AzureWizardExecuteStep<IBuildImageInAzureCon
             const image = outputImages[0];
             context.image = `${image.registry}/${image.repository}:${image.tag}`;
         } else {
-            throw new Error(localize('noImagesBuilt', 'Failed to build image.'));
+            const contentProvider = getContentProvider();
+            await contentProvider.openImageLog(context);
```

Comment:
We should add a button that says "View logs" that lets the user control if the log comes up or not. I think it'll be kind of jarring to have a log file just pop up immediately after an error happens.

Probably can't do this now, but it's a shame once they close the log (or if they don't click it) there's no way for the user to open it later. Definitely seems like a good use-case for the sub-activities thing.